packages/block-library/src/page-list/edit.js

```diff
@@ -185,13 +187,6 @@ export default function PageListEdit( {
 		}
 	};
 
-	useEffect( () => {
-		__unstableMarkNextChangeAsNotPersistent();
-		if ( blockList ) {
-			replaceInnerBlocks( clientId, blockList );
-		}
-	}, [ clientId, blockList ] );
-
```

Comment:
We should also be sure that this change doesn't break any of the checking we did in https://github.com/WordPress/gutenberg/pull/46223